src/Iteration.cpp

```diff
@@ -118,6 +118,13 @@ Iteration::flushFileBased(std::string const& filename, uint64_t i)
         IOHandler->enqueue(IOTask(this, pCreate));
     } else
     {
+      if (IOHandler->accessTypeFrontend == AccessType::CREATE)
```

Comment:
if we can somehow determine that the "old" series has been read at least once, we can also try:
```suggestion
      if (IOHandler->accessTypeFrontend == AccessType::CREATE || IOHandler->accessTypeFrontend == AccessType::READ_WRITE)
```

I would generally be more comfortable if we can place this backend-specific work-around in the ADIOS1IOHandlerImpl file(s) instead of here, we could e.g. return the `OPEN_FILE` call that is triggered here.

(And probably have to add a work-around (skip-if-not-open) to the `OPEN_PATH`, too.)

Reply:
I think read_write should be allowed to read. That's why I did not include it. 

On the other hand, adios does have have both read/write mode

Reply:
you could add
```suggestion
      if (IOHandler->accessTypeFrontend == AccessType::CREATE || IOHandler->backend() != "MPI_ADIOS1")
```
as a really directed frontend hack (for a backend problem).

Reply:
As discussed on Slack, the most general solution to this problem is likely to open files directly in the backend instead of delaying their open.
That means `createFile` should open files already instead of waiting for `openFile`, `writeDataset`, an `writeAttribute`.

Reply:
OK added